### ISSUE TITLE
Some Typography styles might be too general

### ISSUE THREAD

**Issue by MichaelAllenWarner** [utkwdn/utk-atlas#99]
Some of [these base-element rules](https://github.com/utkwdn/utk-atlas/blob/reactify-components/src/scss/framework/styles/_typography.scss#L56-L83) may be more general than intended. The reason I think so is the following discrepancy I'm seeing between accordions previewed in WordPress and accordions in Storybook (from my local branch, where I'm integrating React-Bootstrap):

![Screen Shot 2022-09-08 at 2 43 56 PM](https://user-images.githubusercontent.com/44578578/189201408-e466f2ce-c69f-4bd5-b0b8-f9c4945e68de.png)

![Screen Shot 2022-09-08 at 3 06 32 PM](https://user-images.githubusercontent.com/44578578/189205770-685ee22a-cde7-4617-99e3-c33159b86929.png)

I've traced the discrepancy to the `h2` top-margin rule in the Next.js app, which—judging by what I'm seeing in WordPress—doesn't seem to have a direct counterpart in the framework. When I nix that rule in the dev-tools in Storybook, the accordion looks right.

**Comment by spworky** [comment]
Yes, I am aware of the margin issue. I added generic margins to the main headers, and have not yet trickled that down through the rest of the site. Definitely needs to be looked at.
